api/pytest.ini

```diff
@@ -10,6 +10,8 @@ addopts=
     # FIXME (jsdupuis, 2022-04-14): can be removed when we use flask-sqlalchemy >=2.5
     # (https://github.com/pallets-eco/flask-sqlalchemy/commit/3565f0587519168c5ea4301f6e4bfba8c2ac4dee)
     -W"ignore:'__ident_func__' is deprecated and will be removed in Werkzeug 2.1.:DeprecationWarning"
+    # FIXME (rpaoloni, 2022-04-19): remove deprecation warning for setup of dependecies
```

Comment:
Je verrais bien un message plus précis. Ici, à part toi (et peut-être moi, malgré ma mémoire de poisson rouge !), j'ai peur que peu de gens comprennent le commentaire dans quelques semaines. :) Suggestion : 

    # FIXME (rpaoloni, 2022-04-19): some of our dependencies (swift and rq) use `distutils.version.StrictVersion`,
    # which emits a deprecation warnings with setuptools >=59.6.0. Remove this when we update or remove those
    # dependencies.